src/main/java/co/inventorsoft/academy/jdbc/MyJDBC.java

```diff
@@ -0,0 +1,76 @@
+package co.inventorsoft.academy.jdbc;
+
+
+import lombok.NonNull;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDate;
+@Component
```

Comment:
```suggestion
import java.time.LocalDate;

@Component
```

Keep single empty line after import section.